shuttle/templates/driver_logs/load_logs.html

```diff
@@ -0,0 +1,217 @@
+                    <div class="card-body log-body">
+                        <table class="table table-hover">
+                            <thead>
+                            <tr>
+                                <th scope="col">Driver</th>
+                                <th scope="col">User</th>
+                                <th scope="col">Picked Up</th>
+                                <th scope="col">Dropped Off</th>
+                                <th scope="col">Deleted At</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                {% for i in range(deleted_requests|length) %}
+                                    <td class="driver">
+                                        {% if deleted_requests[i]['driver_name'] != None %}
+                                            {{ deleted_requests[i]['driver_name'] }}
+                                        {% endif %}
+                                    </td>
+                                    <td class="user">
+                                        {% if deleted_requests[i]['user_name'] != None %}
+                                            {{ deleted_requests[i]['user_name'] }}
+                                        {% endif %}
+                                    </td>
+                                    <td class="pick-up-location">
+                                        {% if deleted_requests[i]['pick_up_location'] != None %}
+                                            {{ deleted_requests[i]['pick_up_location'] }}
+                                        {% endif %}
+                                    </td>
+                                    <td class="drop-off-location">
+                                        {% if deleted_requests[i]['drop_off_location'] != None %}
+                                            {{ deleted_requests[i]['drop_off_location'] }}
+                                        {% endif %}
+                                    </td>
+                                    <td class="deleted">
+                                        {% if deleted_requests[i]['completed_at'] != None %}
+                                            {{ deleted_requests[i]['completed_at'] }}
+                                        {% endif %}
+                                    </td>
+                                    </tr>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+        {% endif %}
+    </div>
+
+    <div class="center-page" style="padding-top: 1em">
+        {% if not shuttle_logs and not break_logs and not completed_requests and not deleted_requests %}
+            <h4>There are no Driver logs available for today's date</h4>
+        {% endif %}
+    </div>
+
+{% endblock %}
```

Comment:
Could you make the scrollbars for all of these only appear when necessary?

Reply:
I'm pretty sure that the master branch fixes it but I can do that here too